package.json

```diff
@@ -53,7 +53,7 @@
   },
   "theiaPluginsDir": "plugins",
   "theiaPlugins": {
-    "vscode-builtin-extensions-pack": "https://open-vsx.org/api/eclipse-theia/builtin-extension-pack/1.77.0/file/eclipse-theia.builtin-extension-pack-1.77.0.vsix",
+    "vscode-builtin-extensions-pack": "https://open-vsx.org/api/eclipse-theia/builtin-extension-pack/1.79.0/file/eclipse-theia.builtin-extension-pack-1.79.0.vsix",
```

Comment:
Why 1.79?

Reply:
Because there is no newer version on open-vsx and 1.79 is used in the theia main repository
https://open-vsx.org/extension/eclipse-theia/builtin-extension-pack
https://github.com/eclipse-theia/theia/blob/273c7e269015c3aa8328f364a1475a96dcd625c6/package.json#L105